channel-manager-api.md

```diff
@@ -209,6 +201,157 @@ Mews automatically sends changes in Inventory \(once connection is set up\). Dat
 | `7` | Closed to departure |
 | `8` | Closed to stay |
 
+When all restrictions removed, state |`1`| is sent.
+
+
+```javascript
+       {
+           "ratePlanCode": "r",
+           "spaceTypeCode": "JST",
+           "state": [
+               1
+           ],
+           "minLos": null,
+           "maxLos": null,
+           "from": "2020-09-24",
+           "to": "2020-10-31"
+       }
+```
+
+New restrictions always override old restrictions. State |`1`| is not sent to remove old restrictions, if they were modified.
+
+State |`2`| is always sent in combination with state |`6`|, |`7`|, or |`8`|, or all together.
+
+```javascript
+        {
+            "ratePlanCode": "Ff",
+            "spaceTypeCode": "DEL",
+            "state": [
+                2,
+                8,
+                6,
+                7
+            ],
+            "minLos": null,
+            "maxLos": null,
+            "from": "2020-09-24",
+            "to": "2020-10-31"
+        }
+```
+Closed to Stay
```

Comment:
is it a title or what does this line mean?

Reply:
its a title. I corrected it to ##### format